src/model_locator.py

```diff
@@ -0,0 +1,25 @@
+import os
+import dotenv
+
+
+def __load_model_env(model: str) -> str:
+    dotenv.load_dotenv()
+
+    path_from_env_file = os.getenv(model)
+    if path_from_env_file is not None:
```

Comment:
Does it matter if the `.env` file uses a relative path to a file located in a subdir?

E.g., `MODEL_LABELING = ./model/cira-labeler.ckpt`

I prefer a convention-over-configuration approach to this problem.
Maybe there is a filter for the development container that would allow not mounting the `.env` file into the container.

Reply:
We should refactor the is-in-dev-container behavior in another PR. 

**This PR is required** because the inconsistent env loading in the prod and test environment blocks other types of refactoring, such as the pre-loading of `BertModel`.

Reply:
I agree that there might be a more elegant solution. For now, the `isfile` check should be sufficient.